src/controllers/auth/forgotPassword.ts

```diff
@@ -45,8 +49,26 @@ export const postForgot = asyncHandler(
       throw new InternalServerError()
     }
 
-    // TODO: email the token to the user/guest, return a 204 response
-    logger.info({ token })
-    throw new NotImplemented()
+    try {
+      const urlPath = urljoin(
+        env<string>(EnvVariable.APP_URL, 'http://localhost:3000'),
+        env<string>(EnvVariable.AUTH_PASSWORD_RESET_PATH, '/password/reset'),
+      )
+
+      await mail.send({
+        from: processEnv(EnvVariable.MAIL_FROM_ADDRESS),
+        to: { name: user.name, address: user.email },
+        subject: env<string>(EnvVariable.FORGOT_PASSWORD_SUBJECT, 'Reset Password'),
+        ...mail.render(MailTemplate.forgotPassword, {
+          appName: processEnv(EnvVariable.APP_NAME),
+          title: env<string>(EnvVariable.FORGOT_PASSWORD_TITLE, 'Reset Password'),
+          url: `${urlPath}?${stringify({ token, userId: user.id })}`,
+        }),
+      })
+      return res.status(204).send()
+    } catch (error) {
+      logger.error(`Unable to save password reset, error: ${error}`)
```

Comment:
I didn't mean to update the message to `Unable to save password reset` It should still be what is was:
```suggestion
      logger.error(`Unable to send email, error: ${error}`)
```

Reply:
Lol, didn't even realise that